src/protocol/java/src/main/java/org/apache/jmeter/protocol/java/sampler/JavaSampler.java

```diff
@@ -121,9 +121,9 @@ private void initClass() {
             javaClass = Class.forName(name, false, Thread.currentThread().getContextClassLoader());
             Method method = javaClass.getMethod("teardownTest", JavaSamplerContext.class);
             isToBeRegistered = !method.getDeclaringClass().equals(AbstractJavaSamplerClient.class);
-            log.info("Created class: {}. Uses tearDownTest: ", name, isToBeRegistered);
+            log.info("Created class: {}. Uses tearDownTest: {}", name, isToBeRegistered);
         } catch (Exception e) {
-            log.error("{}\tException initialising: ", whoAmI(), name, e);
+            log.error("{}\tException initialising: {} {}", whoAmI(), name, e.getMessage());
```

Comment:
The exception can be the last parameter without a placeholder, and it will render as a stacktrace, so `.getMessage` is not needed here.

```suggestion
            log.error("{}\tException initialising: {}", whoAmI(), name, e);
```